packages/mobile/src/utils/image.ts

```diff
@@ -32,3 +40,7 @@ export const saveProfilePicture = async (dataUrl: string): Promise<string> => {
     `file://${RNFS.DocumentDirectoryPath}/profile-${Date.now()}`
   )
 }
+
+export const saveRecipientPicture = async (dataUrl: string, address: string): Promise<string> => {
+  return saveImageDataUrlToFile(dataUrl, `file://${RNFS.DocumentDirectoryPath}/${address}`)
```

Comment:
Looks like we should store them in a dedicated folder.
This way we can more easily clean up pictures if needed, or list them.

Reply:
Not sure what you're suggesting here but our structure needs to be compatible with CIP8.

`<BUCKET>/<WHATEVER_PATH>/<USER_ADDRESS>/account/picture`.

Reply:
@AlexBHarley this is the path for where the pictures are stored internally in Valora after fetching them from the CIP8 backend